tableview/src/main/java/com/evrencoskun/tableview/handler/ISelectableModel.java

```diff
@@ -0,0 +1,12 @@
+package com.evrencoskun.tableview.handler;
+
+import com.evrencoskun.tableview.adapter.recyclerview.holder.AbstractViewHolder;
+
+/**
+ * Created by cedricferry on 8/2/18.
+ */
+
+public interface ISelectableModel {
```

Comment:
If you can, it'd be nice to have the method of this interface annotated with `@NonNull`/`@Nullable`, based on your needs.